### ISSUE TITLE
Bulk forecast and re-forecast

### ISSUE THREAD

**Issue by okms** [Puzzlepart/did#606]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
Add and remove forecasts in bulk, e.g. per quarter

### Description
from [Teams thread](https://teams.microsoft.com/l/message/19:f936be46d9134b8191421604341699d6@thread.tacv2/1603089715962?tenantId=dcb89c10-67b8-48cc-8819-85de37e8af91&groupId=46adb6c8-8c09-409f-a333-2d131ab6aff8&parentMessageId=1603089715962&teamName=Did&channelName=Forcasting%20Beta&createdTime=1603089715962) machine translated
It was time consuming to print the forecast for each individual week, and I know that there will be many changes continuously. If I get an extra customer that I will spend a certain amount of time on each week, then today I have to un-forecast and forecast again every week. I would like to have a button where I can, for example, forecast the coming quarter. Then we also set expectations that everyone will have updated the coming quarter in their calendar (for those of us who have an idea of ​​how they are allocated in January, the rest of us can forecast it empty, it also has a value).
